pkg/rbuser/ldap.go

```diff
@@ -0,0 +1,27 @@
+package rbuser
+
+import (
+	"fmt"
+
+	ldap "gopkg.in/ldap.v2"
+)
+
+// LdapConf Server object used for connecting to server
+type LdapConf struct {
```

Comment:
I feel like this should have only one `New` function declaration. It is also being created in `internal/pkg/rbldap/ldap.go` using `cli.Context`. Ideally that package should call `NewLdapConf` which is defined here. That way these variables in the struct don't need to be public, because they are never used outside `LdapConf`'s functions.

Reply:
this doesn't really make sense this doesn't have any new function calls